fwprovider/resource_options.go

```diff
@@ -414,6 +550,108 @@ func (r *clusterOptionsResource) Schema(
 				Description: "Cluster wide migration network CIDR.",
 				Optional:    true,
 			},
+			"next_id": schema.SingleNestedAttribute{
+				Attributes: map[string]schema.Attribute{
+					"lower": schema.Int64Attribute{
+						Description: "The minimum number for the next free VM ID.",
+						MarkdownDescription: "The minimum number for the next free VM ID. " +
+							fmt.Sprintf("Must be higher or equal to %d", ClusterOptionsNextIDLowerMinimum),
+						Optional: true,
+						Validators: []validator.Int64{
+							int64validator.AtLeast(ClusterOptionsNextIDLowerMinimum),
+						},
+					},
+					"upper": schema.Int64Attribute{
+						Description: "The maximum number for the next free VM ID.",
+						MarkdownDescription: "The maximum number for the next free VM ID. " +
+							fmt.Sprintf("Must be less or equal to %d", ClusterOptionsNextIDLowerMaximum),
+						Optional: true,
+						Validators: []validator.Int64{
+							int64validator.AtMost(ClusterOptionsNextIDLowerMaximum),
+						},
+					},
+				},
+				Description: "The ranges for the next free VM ID auto-selection pool.",
+				Optional:    true,
+			},
+			"notify": schema.SingleNestedAttribute{
+				Attributes: map[string]schema.Attribute{
+					"ha_fencing_mode": schema.StringAttribute{
+						Description:         "Cluster-wide notification settings for the HA fencing mode.",
+						MarkdownDescription: "Cluster-wide notification settings for the HA fencing mode. Must be `always` | `never`.",
+						Optional:            true,
+						Validators: []validator.String{
+							stringvalidator.OneOf(
+								[]string{
+									"always",
+									"never",
+								}...,
+							),
+						},
+					},
+					"ha_fencing_target": schema.StringAttribute{
+						Description:         "Cluster-wide notification settings for the HA fencing target.",
+						MarkdownDescription: "Cluster-wide notification settings for the HA fencing target.",
+						Optional:            true,
+						Validators: []validator.String{
+							stringvalidator.UTF8LengthAtLeast(1),
+							stringvalidator.RegexMatches(regexp.MustCompile(`^\S|^$`), "must not start with whitespace"),
+							stringvalidator.RegexMatches(regexp.MustCompile(`\S$|^$`), "must not end with whitespace"),
+						},
```

Comment:
This validation is duplicated few times in the schema, so could extracted in a shared validator.